builtin/providers/circonus/resource_circonus_check_http_test.go

```diff
@@ -0,0 +1,180 @@
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.1136493216.unit", "seconds"),
+
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.active", "true"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.name", "tt_connect"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.tags.#", "4"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.tags.30226350", "app:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.tags.3219687752", "app:jezebel"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.tags.1543130091", "lifecycle:unittests"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.tags.3241999189", "source:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.type", "numeric"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.unit", "milliseconds"),
+
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.active", "true"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.name", "tt_firstbyte"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.tags.#", "4"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.tags.30226350", "app:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.tags.3219687752", "app:jezebel"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.tags.1543130091", "lifecycle:unittests"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.tags.3241999189", "source:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.type", "numeric"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.unit", "milliseconds"),
+
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "tags.#", "4"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "tags.30226350", "app:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "tags.3219687752", "app:jezebel"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "tags.1543130091", "lifecycle:unittests"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "tags.3241999189", "source:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "target", "127.0.0.1"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "type", "http"),
+				),
+			},
+		},
+	})
+}
+
+const testAccCirconusCheckHTTPConfig = `
+variable "http_check_tags" {
+  type = "list"
+  default = [ "app:circonus", "app:jezebel", "lifecycle:unittests", "source:circonus" ]
+}
+
+resource "circonus_metric" "status_code" {
+  name = "code"
```

Comment:
Nope, these are local and won't conflict with anything.  Internally `circonus_metrics` are actually random UUIDs.  To be safe, I checked and don't see any metrics with the same `name`.  To the best of my knowledge, these tests can be run in parallel without issue (that's how I've been testing this code to date).  I think if there were two concurrent instances of `make test` running for this provider it could cause a problem, but I'm not sure it's worth solving.  At present, a full test run looks like (run only a few seconds ago):

```
==> Checking that code complies with gofmt requirements...
go generate $(go list ./... | grep -v /terraform/vendor/)
2017/03/03 16:10:38 Generated command/internal_plugin_list.go
TF_ACC=1 go test ./builtin/providers/circonus -v  -timeout 120m
=== RUN   TestAccDataSourceCirconusAccount
--- PASS: TestAccDataSourceCirconusAccount (2.03s)
=== RUN   TestAccDataSourceCirconusCollector
--- PASS: TestAccDataSourceCirconusCollector (1.88s)
=== RUN   Test_MetricChecksum
--- PASS: Test_MetricChecksum (0.00s)
=== RUN   TestProvider
--- PASS: TestProvider (0.00s)
=== RUN   TestProvider_impl
--- PASS: TestProvider_impl (0.00s)
=== RUN   TestAccCirconusCheckCAQL_basic
--- PASS: TestAccCirconusCheckCAQL_basic (4.99s)
=== RUN   TestAccCirconusCheckCloudWatch_basic
--- PASS: TestAccCirconusCheckCloudWatch_basic (5.78s)
=== RUN   TestAccCirconusCheckHTTP_basic
--- PASS: TestAccCirconusCheckHTTP_basic (5.75s)
=== RUN   TestAccCirconusCheckHTTPTrap_basic
--- PASS: TestAccCirconusCheckHTTPTrap_basic (8.14s)
=== RUN   TestAccCirconusCheckICMPPing_basic
--- PASS: TestAccCirconusCheckICMPPing_basic (5.51s)
=== RUN   TestAccCirconusCheckJSON_basic
--- PASS: TestAccCirconusCheckJSON_basic (9.56s)
=== RUN   TestAccCirconusCheckMySQL_basic
--- PASS: TestAccCirconusCheckMySQL_basic (5.06s)
=== RUN   TestAccCirconusCheckPostgreSQL_basic
--- PASS: TestAccCirconusCheckPostgreSQL_basic (5.42s)
=== RUN   TestAccCirconusCheckTCP_basic
--- PASS: TestAccCirconusCheckTCP_basic (6.34s)
=== RUN   TestAccCirconusContactGroup_basic
--- PASS: TestAccCirconusContactGroup_basic (4.15s)
=== RUN   TestAccCirconusGraph_basic
--- PASS: TestAccCirconusGraph_basic (8.15s)
=== RUN   TestAccCirconusMetric_basic
--- PASS: TestAccCirconusMetric_basic (0.02s)
=== RUN   TestAccCirconusMetric_tagsets
--- PASS: TestAccCirconusMetric_tagsets (0.05s)
=== RUN   TestAccCirconusStreamGroup_basic
--- PASS: TestAccCirconusStreamGroup_basic (3.64s)
=== RUN   TestAccCirconusTrigger_basic
--- PASS: TestAccCirconusTrigger_basic (7.97s)
PASS
ok      github.com/hashicorp/terraform/builtin/providers/circonus       84.459s
```